README.md

```diff
@@ -38,13 +41,156 @@ class Translations {
 }
 ```
 
-### Flavors
+Did you notice the `@GenerateArb()` annotation? That where the magic come from.
+
+Once you run you favorite build_runner command it will generate the ARB corresponding to this class.
```

Comment:
Once you run your favourite `build_runner` command it will generate the ARB corresponding to this class.